releasenotes/notes/two-step-transpile-f20d709a7a0c42dd.yaml

```diff
@@ -0,0 +1,66 @@
+---
+features:
+  - |
+    Allow two transpiler stages in the :class:`~qiskit.utils.QuantumInstance`, one for
+    parameterized circuits and a second one for bound circuits (i.e. no free parameters) only.
+    If a quantum instance with passes for unbound and bound circuits is passed into a
+    :class:`~qiskit.opflow.CircuitSampler`, the sampler will attempt to apply the unbound pass
```

Comment:
I'd probably have split this into a separate note, but it's fine integrated into this too.